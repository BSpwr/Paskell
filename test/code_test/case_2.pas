program Case2Test;
var
    val: real;
begin

   val := 19;

    case (val < 20) of
      true: writeln('Yep, 19 is < 20' );
      false : writeln('Better try again' );
    else
      writeln('Should never run');
   end;

end.