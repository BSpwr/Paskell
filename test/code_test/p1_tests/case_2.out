Yep, 19 is < 20
